pkg/prowgen/prowgen_konflux.go

```diff
@@ -341,6 +345,44 @@ func GenerateKonfluxServerlessOperator(ctx context.Context, openshiftRelease Rep
 	return nil
 }
 
+func generateFBCApplications(soMetadata *project.Metadata, openshiftRelease Repository, r Repository, branch string, release string, resourceOutputPath string, buildArgs []string) error {
+	soMetadata.Requirements = &project.Requirements{
+		OcpVersion: project.OcpVersion{
+			List: []string{
+				"4.13",
+				"4.17",
+			},
+		},
+	}
```

Comment:
This is only temporary for testing until https://github.com/openshift-knative/serverless-operator/pull/2926/ is in